dev/com.ibm.ws.logging_2_fat/test-applications/logger-servlet/resources/.gitignore

```diff
@@ -0,0 +1 @@
+/WEB-INF
```

Comment:
Is this necessary?

Reply:
I see other test-apps have it, so I'll leave it for now.